src/fsharp/fsi/fsi.fs

```diff
@@ -2000,31 +2064,39 @@ let internal SpawnInteractiveServer
              Server.Shared.FSharpInteractiveServer.StartServer(fsiOptions.FsiServerName,server)
          with e ->
              fprintfn fsiConsoleOutput.Error "%s" (FSIstrings.SR.fsiExceptionRaisedStartingServer(e.ToString())))
-  
+#endif
 
-let internal StartStdinReadAndProcessThread
-                                  (lcid, istateRef, errorLogger, 
-                                   fsiConsoleInput: FsiConsoleInput, 
-                                   fsiConsoleOutput: FsiConsoleOutput,
-                                   fsiStdinLexerProvider: FsiStdinLexerProvider, 
-                                   fsiInteractionProcessor : FsiInteractionProcessor,
-                                   exitViaKillThread) = 
+let internal StartStdinReadAndProcessThread(
+#if FX_LCIDFROMCODEPAGE
+                                            lcid,
+#endif
+                                            istateRef, 
+                                            errorLogger, 
+                                            fsiConsoleInput: FsiConsoleInput, 
+                                            fsiConsoleOutput: FsiConsoleOutput,
+                                            fsiStdinLexerProvider: FsiStdinLexerProvider, 
+                                            fsiInteractionProcessor: FsiInteractionProcessor,
+                                            exitViaKillThread) = 
     if !progress then fprintfn fsiConsoleOutput.Out "creating stdinReaderThread";
     let cont = ref Completed 
     let tokenizerRef = ref (fsiStdinLexerProvider.CreateStdinLexer())
+#if FX_LCIDFROMCODEPAGE
```

Comment:
I doubt if we will have a version of the compiler that runs against the desktop CLR.  Therefore all of that code will likely go away.